.github/workflows/lint.yml

```diff
@@ -9,25 +9,17 @@ on:
       - main
 
 jobs:
-  commitlint:
+  trunk:
+    if: >
```

Comment:
maybe we can drop this `if` check for now, we don't really have the `skip workflows` label at the moment, if this is needed, we can always add it in future